thirdeye/thirdeye-frontend/app/pods/components/anomaly-graph/component.js

```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
```

Comment:
@yee-lkd those files were actually generated with `ember g`. This is the pod structure default's naming convention i.e. `components/component-name/template.hbs` `components/component-name/component.js`